built-ins.c

```diff
@@ -0,0 +1,22 @@
+#include "main.h"
+
+/**
+ * handle_builtins - handles the built in commands
+ * @args: command
+ * @env: env vars
+ */
+
+void handle_builtins(char **args, char **env)
+{
+	if (_strcmp(args[0], "exit"))
+	{
+		if (args[1] != NULL)
+			exit(_atoi(args[1]));
+		else
+			exit(0);
+	}
+
+	if (_strcmp(args[0], "env"))
```

Comment:
probably should print an error if for example command is `env blabla`, the err msg is `env: 'blabla': No such file or directory`, u don't need to test for `blabla` if it's a valid file or dir, i tried it in my shell and whatever the second arg is it prints the same err